dev/io.openliberty.jakarta.expressionLanguage.5.0/resources/jakarta/el/LocalStrings.properties

```diff
@@ -0,0 +1,52 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Is this from the expressionLanguage.4.0?

Reply:
No, it's from Tomcat's expression language 5.0 api source